bugrap/src/main/java/org/vaadin/bugrap/ui/LoginModel.java

```diff
@@ -4,11 +4,16 @@
 import org.vaadin.bugrap.DatabaseHelper;
 import org.vaadin.bugrap.domain.entities.Reporter;
 
+import com.vaadin.navigator.Navigator;
 import com.vaadin.ui.Notification;
 import com.vaadin.ui.Notification.Type;
 
 public class LoginModel extends BaseModel{
 
+	public LoginModel(Navigator navigator) {
+		super(navigator);
+	}
+	
 	public void login(String un, String pw) {
 		Reporter reporter = getRepository().authenticate(un, pw);
 		if (reporter == null && DatabaseHelper.initializeIfEmpty(getRepository())) {
```

Comment:
1) `System.out.println` is evil ;D
Use it for your debugging only. If you want to log better use a real Logger api.

2) There is still a better way of populating the DB. Check for servlet lifecycle events.